components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -422,7 +540,6 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
    real(r8) :: ideep_crm(pcols)                    ! gathering array for convective columns
    logical  :: ls, lu, lv, lq(pcnst)               ! flags for updating ptend
```

Comment:
Just a cosmetic suggestion. Do you need to keep ls, lu, lv and lq? there are only being used once in ptend_init subroutine. they are all set to true. We could just put 'true' directly in the argument section.

Reply:
Good point, this does seem unnecessary. I'll fix it.